xapian-core/cluster/cluster.cc

```diff
@@ -296,9 +296,9 @@ Point::Point(const TermListGroup &tlg, const Document &document_)
 	string term = *it;
 	double termfreq = tlg.get_termfreq(term);
 
-	/// If term frequency of the term is lesser than or equal
-	/// to 1 or wdf is lesser than 1, we do not consider this
-	/// term for document vector calculations
+	// If the term doesn't exist within the document or
+	// exists in only one document within the MSet, then the
+	// terms are not used for document vector calculations
```

Comment:
We call them boolean terms or filter terms - I explained that above: "filter terms (terms added with `add_boolean_term()`)"

Although they aren't part of the document text, they are perhaps still be of interest when clustering - e.g. clusters based on (or partly based on) things like the file type, where it is on disk, when it was created, etc.

At least for now it's a reasonable restriction to ignore them, and that wording seems OK except I'd lose the "or metadata term" bit.